### ISSUE TITLE
Version of the ansible operator image with newer git

### ISSUE THREAD

**Issue by willthames** [operator-framework/operator-sdk#1394]
## Feature Request

**Is your feature request related to a problem? Please describe.**
git-sync relies on worktrees to provide git repos. Centos 7's git does not seem to understand worktrees.

**Describe the solution you'd like**
I would like the ansible-operator image to contain a newer version of git (I don't know how new to be honest)

An interim fix for me that works is

```
FROM quay.io/operator-framework/ansible-operator:v0.7.0 AS operator

FROM alpine:3.9
COPY --from=operator /usr/local/bin /usr/local/bin
ENTRYPOINT /usr/local/bin/entrypoint
RUN apk add --no-cache git ansible python3
RUN apk add --no-cache -t .build-deps gcc musl-dev python3-dev linux-headers && \
    python3 -m pip install ansible-runner jmespath && \
    apk del .build-deps
RUN apk add --no-cache bash
USER 1001
ENV RUNNER_BASE_COMMAND ansible-playbook
ENV OPERATOR /usr/local/bin/ansible-operator
ENV USER_UID 1001
ENV USER_NAME ansible-operator
ENV HOME /opt/ansible
```

but I'd really rather not maintain my own image, it'll only get out of date.

**Comment by camilamacedo86** [comment]
Hi @willthames,

I am afraid that I could not understand what exactly you are looking for. Note that you can run `operator-sdk build <image>:tag` and your docker file will be created then you just need to push it.  

Could you please describe what, when and why you are looking for? I will be very glad you with.

**Comment by hasbro17** [comment]
@camilamacedo86 I think @willthames is asking for a newer version of git than what we currently have in the Ansible operator base image for v0.7.0.
https://quay.io/repository/operator-framework/ansible-operator?tab=tags

`operator-sdk build ...` and `docker push` are used to build the operator image on top of that base image.

Deferring to the ansible-operator folks on this.
/cc @shawn-hurley @fabianvf

**Comment by flickerfly** [comment]
I believe the ansible operator image resolved this in version 0.10.0. 
https://quay.io/repository/operator-framework/ansible-operator?tab=tags
